access_scopes.go

```diff
@@ -0,0 +1,23 @@
+package goshopify
+
+type AccessScope struct {
+	Handle string `json:"handle,omitempty"`
+}
+
+type AccessScopeResource struct {
+	AccessScopes []AccessScope `json:"access_scopes"`
+}
+
+type AccessScopeService interface {
+	List(options interface{}) ([]AccessScope, error)
+}
+
+type AccessScopeServiceOp struct {
+	client *Client
+}
+
+func (a *AccessScopeServiceOp) List(options interface{}) ([]AccessScope, error) {
```

Comment:
This [takes no arguments](https://shopify.dev/docs/admin-api/rest/reference/access/accessscope#index-2021-04) but there are other cases like this and `options interface{}` is still used so, I added it.